src/app/zap-templates/zcl/bridged-device-basic.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+    <domain name="CHIP" spec="chip-0.7" dependsOn="zcl-1.0-07-5123-03" certifiable="true"/>
+    <cluster>
+        <name>Bridged Device Basic</name>
+        <domain>CHIP</domain>
+        <description>Attributes for determining basic information about a bridged device, and setting user device information such as location.</description>
+        <code>0x0039</code>
+        <define>BRIDGED_DEVICE_BASIC_CLUSTER</define>
+        <client tick="false" init="false">true</client>
+        <server tick="false" init="false">true</server>
+        <globalAttribute side="either" code="0xFFFD" value="3"/>
+
```

Comment:
where does the interactionModelVersion attribute get declared?

Reply:
InteractionModelVersion is still declared in the BASIC cluster. According to the spec, the derived "Bridged Device Basic" cluster does not carry over this attribute. As such, the IM Version would be retrieved from the Basic cluster (which would be on EP0 of the bridge)